latch/registry/table.py

```diff
@@ -373,6 +389,8 @@ class TableUpdate:
 
     table: Table
 
+    # upsert record
```

Comment:
These boundary comments are unnecessary

Reply:
following the pattern set by max elsewhere in this api, idrc either way but it doesnt hurt

Reply:
well I had them around because the overload signatures take a full screen